sopel_modules/youtube/youtube.py

```diff
@@ -37,12 +43,29 @@ class YoutubeSection(StaticSection):
     api_key = ValidatedAttribute('api_key', default=NO_DEFAULT)
     """The Google API key to auth to the endpoint"""
 
+    color_logo = ValidatedAttribute("color_logo", bool, default=True)
+    """Color the YouTube text like the YouTube logo"""
+
+    info_items = ListAttribute(
+        "info_items",
+        default=["uploader", "date", "length", "views"],
```

Comment:
I'd put these in a different order, but eh. Take it or leave it.

```suggestion
        default=["length", "uploader", "views", "date"],
```